### ISSUE TITLE
Should we create branches to separate the front-end and back-end?

### ISSUE THREAD

**Issue by matthewcoupal** [SUNYPotsdamACM/SUNYPotsdamACM.github.io#1]
I think this might be nice, so that we aren't mixing all of our commits to the same place.

I'm thinking this would be a good setup for branches:
- master
- front-end
- back-end
- features
  @JessePeplinski and @fuzzn8r , let me know what you guys think.

**Comment by fuzzn8r** [comment]
I think that's fine so long as we remember to merge responsibly :)

**Comment by JessePeplinski** [comment]
I agree, this should work well. I think that we might be able to incorporate a branch for dev builds as well, which we can figure out next Tuesday.

**Comment by matthewcoupal** [comment]
So, shall I make the branches now, or should I wait to make them so we can discuss the branching setup as a collective Tuesday?

**Comment by fuzzn8r** [comment]
I'd say if you currently have code to commit to any of those branches, go ahead now. Otherwise I say wait until the next meeting.

**Comment by matthewcoupal** [comment]
Okay, I might work on it over the break. If I do, I will just make the branches on my local machine and wait to push until the next meeting.

**Comment by fuzzn8r** [comment]
I got a better idea: 

Go ahead and start working on whatever branch you feel comfortable working on. We'll use future meetings to discuss the branch(es) and also discuss merging.
